crates/topos-sequencer/src/lib.rs

```diff
@@ -64,11 +73,16 @@ pub async fn launch(
         }
     };
 
+    // TODO Here determine if websocket endpoint is provided or should be derived from http endpoint
```

Comment:
Do you plan to address this in this PR or in a separate ticket?

Reply:
This is outdated TODO